CHANGELOG.md

```
@@ -1,3 +1,12 @@
+## v11.0.0
+April 20 2026
+
+#### Added
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)